sui_programmability/examples/games/sources/SharedTicTacToe.move

```diff
@@ -93,9 +92,9 @@ module Games::SharedTicTacToe {
         let cell = Vector::borrow_mut(Vector::borrow_mut(&mut game.gameboard, (row as u64)), (col as u64));
         assert!(*cell == MARK_EMPTY, ECELL_OCCUPIED);
 
-        *cell = game.cur_turn;
+        *cell = game.cur_turn % 2;
         update_winner(game);
-        game.cur_turn = if (game.cur_turn == MARK_X) MARK_O else MARK_X;
+        game.cur_turn = game.cur_turn + 1;
```

Comment:
this implementation of rotating `cur_turn` does not help us track the total number of turns that already took place, which is used to check draws. I'm updating it to increments as in the `TicTacToe` version